src/drivers/pwm_out/PWMOut.cpp

```diff
@@ -1862,6 +1906,21 @@ int PWMOut::test()
 	return rv;
 }
 
+int PWMOut::mavlink_servo_test(int servo_id, float value)
```

Comment:
This was a useful test function, but can likely be removed (in favor of the implementation in Commander)